src/demikernel/libos.rs

```diff
@@ -52,18 +52,21 @@ pub enum LibOS {
 impl LibOS {
     cfg_if::cfg_if! {
         if #[cfg(feature = "catnip-libos")] {
+            /// Waits a pending operation in a I/O queue.
```

Comment:
This sentence isn't grammatically correct.  Do you mean "Waits for a pending operation" or "Waits on a pending operation"?  This same comment is repeated 5 times below.

Reply:
I mean "waits on". Thanks.